assets/js/components/user-input/UserInputPreview.js

```diff
@@ -47,18 +48,33 @@ export default function UserInputPreview( props ) {
 		submitChanges,
 		error,
 	} = props;
-
+	const previewContainer = useRef();
 	const settings = useSelect( ( select ) => select( CORE_USER ).getUserInputSettings() );
-
 	const {
 		USER_INPUT_ANSWERS_GOALS,
 		USER_INPUT_ANSWERS_HELP_NEEDED,
 		USER_INPUT_ANSWERS_POST_FREQUENCY,
 		USER_INPUT_ANSWERS_ROLE,
 	} = getUserInputAnwsers();
+	const [ page ] = useQueryArg( 'page' );
+
+	useEffect( () => {
+		if ( page.startsWith( 'googlesitekit-settings' ) ) {
+			return;
+		}
+
+		if ( previewContainer?.current ) {
```

Comment:
Please, use the early return to reduce unnecessary nesting.

Reply:
👍